src/include/hip/rpp_hip_common.hpp

```diff
@@ -80,11 +80,14 @@ struct RPPTensorFunctionMetaData
     }
 };
 
-#define LOCAL_THREADS_X 16
-#define LOCAL_THREADS_Y 16
-#define LOCAL_THREADS_Z 1
-#define ONE_OVER_255 0.00392157f
-#define SIX_OVER_360 0.01666667f
+#define LOCAL_THREADS_X                 16
+#define LOCAL_THREADS_Y                 16
+#define LOCAL_THREADS_Z                 1
+#define ONE_OVER_255                    0.00392157f
+#define SIX_OVER_360                    0.01666667f
+#define XORWOW_COUNTER_INC              0x587C5     // Hex 0x587C5 = Dec 362437U - xorwow counter increment
+#define XORWOW_EXPONENT_MASK            0x3F800000  // Hex 0x3F800000 = Bin 0b111111100000000000000000000000 - 23 bits of mantissa set to 0, 01111111 for the exponent, 0 for the sign bit
+
 #define BYTE_TO_BINARY_PATTERN "%c%c%c%c%c%c%c%c"
 #define BYTE_TO_BINARY(byte)  \
```

Comment:
what is the use of this macro?

Reply:
That one had been added for hip debugging purposes. It helps display U8/F32 pixels in binary. (Could be removed now, or later once more tensor hip kernels are added)